rust/chains/hyperlane-sealevel/src/interchain_gas.rs

```diff
@@ -74,3 +76,11 @@ impl Indexer<InterchainGasPayment> for SealevelInterchainGasPaymasterIndexer {
         Ok(1)
     }
 }
+
+#[async_trait]
+impl NonceIndexer<InterchainGasPayment> for SealevelInterchainGasPaymasterIndexer {
+    async fn nonce_at_tip(&self) -> ChainResult<(u32, u32)> {
+        info!("Gas payment indexing not implemented for Sealevel");
+        Ok((1, 1))
```

Comment:
Returning this for now makes sense to me - https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2501 tracks stitching it together. For now there's no concept of an IGP sequence number / nonce in Ethereum so there's not much else you can do atm